libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -103,9 +127,50 @@ export default function TeamCharacterSelector({
         }
       >
         <CardContent sx={{ display: 'flex', justifyContent: 'center', pb: 0 }}>
-          <Typography variant="h5" display="flex">
-            {team.name}
-          </Typography>
+          <CardActionArea>
```

Comment:
Add the padding here instead
```suggestion
          <CardActionArea sx={{ px: 2, pt: 2 }}>
```